.env

```diff
@@ -1 +1 @@
-ATLAS_URI=mongodb+srv://admin-ronan:meanpie@cluster0-qw2ci.gcp.mongodb.net/test?retryWrites=true&w=majority 
```

Comment:
Revert the ATLAS_URI back to what it was otherwise we can't access the database as supposed to